plugins/jobs/server/utils.py

```diff
@@ -0,0 +1,28 @@
+from girder.api.rest import getApiUrl
+from girder.utility.model_importer import ModelImporter
+
+
+def jobInfoSpec(job, token=None, logPrint=True):
+    """
+    Build the jobInfo specification for a task to write status and log output
+    back to a Girder job.
+
+    :param job: The job document representing the worker task.
+    :type job: dict
+    :param token: The token to use. Creates a job token if not passed.
+    :type token: str or dict
+    :param logPrint: Whether standard output from the job should be
```

Comment:
show be printed?

Reply:
What about stderr?

Reply:
Sorry didn't see it was WIP

Reply:
It is totally fine. Actually after talking to @kotfic , we decided to go with an event based system. I will do some refactoring.